### ISSUE TITLE
Upgrade Github templates

### ISSUE THREAD

**Issue by fflaten** [pester/Pester#1732]
## 1. General summary of the issue

Request to split current issue template on github repo into different templates for e.g. bugs and feature requests.

## 2. Describe Your Environment

Web browser

## 3. Expected Behavior

Dedicated templates for the common scenarios which is shown as a menu when creating a new issue. [Example](https://github.com/PowerShell/PowerShell/issues/new/choose)

- Bug Report
- Feature Request
- General question/support (provide template or just refer to https://github.com/pester/Pester#questions, the discussion issue etc.?)

- Link to slack channel for "community support"?

Maybe also update Pull Request template with Summary, Context and a checklist (remembered meaning title, tests required/added, breaking change yes/no) ?

## 4.Current Behavior

One issue template for all scenarios with a lot of comments 

## 5. Possible Solution

Create new [issue templates suited for different scenarios](https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/configuring-issue-templates-for-your-repository), using the current template as a baseline for content.

## 6. Context

Lots of boilerplate that's not always necessary, but can feel like a "wall of text" for new user.
Simplify issue documentation by providing different templates for e.g. bugs and feature requests. Can include more specific steps and requirements for each scenario.

**Comment by nohwnd** [comment]
Agree about the wall of text. Personally I just delete the template every time, but when it is filled out it actually helps. So at minimum I would like to know what is the Pester version and OS that is being used. 


Do you want to implement this?

**Comment by fflaten** [comment]
Played around with issue templates in a demo repo. See https://github.com/fflaten/githubsandbox/issues/new/choose . 

Wasn't sure where general questions should go. v5 discussion thread, twitter and slack are mentioned in the bottom of README, while stack overflow, gitter and slack are mentioned on the [website](https://pester.dev/docs/contributing/introduction#where-to-get-support). As a first draft, I added a link to the website and repurposed the current issue template for questions (blank issue link at the bottom).

Thoughts?

Pull request coming up later.

**Comment by nohwnd** [comment]
I see few things. When you PR it I will comment on that. I see few things there.

**Comment by fflaten** [comment]
Let me rephrase that last sentence, pull request **template** coming ut later. 😄 
Or should we discuss that template in it's own PR?
